cpu/fe310/periph/pm.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Ken Rabold
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_fe310
+ * @{
+ *
+ * @file        pm.c
+ * @brief       Implementation of the CPU power management for SiFive FE310
+ *
+ * @author      Ken Rabold
+ * @}
+ */
+
+#include "periph/pm.h"
+
+void pm_set_lowest(void)
+{
+    /* __asm__("wfi"); */
+}
+
+void pm_off(void)
+{
+}
+
+void pm_reboot(void)
+{
```

Comment:
These should be implemented, not empty.

Reply:
Unfortunately the fe310 CPU doesn't have a SW reboot capability

Reply:
How about power modes?

Reply:
It does have a lower power mode that is entered with WFI, but calling that suspends the interrupts for the debugger, which makes it difficult to debug. That is why WFI is commented out.  We could add additional comments to that effect to the source